### PR TITLE
Makes towels digitigrade-compatibles

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
They previously looked awkward with digitigrade legs, and even straight-up made the synthetic humanoid digitigrade legs go invisible because they weren't setting the `supports_variations_flags` at all. I'm fixing the issue of them going invisible to begin with in a separate PR.

## How This Contributes To The Skyrat Roleplay Experience
Wearing towels as a digitigrade now will look ever so slightly less awkward.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image
image

</details>

## Changelog

:cl: GoldenAlpharexx
fix: Towels will now properly display on digitigrade characters.
imageadd: Towels now have a digitigrade worn sprite too!
/:cl:

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]

File: modular_skyrat/master_files/code/modules/clothing/towels.dm [hunk 1]
```diff
@@ -38,6 +41,7 @@
 	item_flags = NOBLUDGEON
 	resistance_flags = FLAMMABLE
 	flags_inv = SHOWSPRITEEARS // Only relevant when in head shape, but useful to keep around regardless.
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
```
Comment (ghost) [inline modular_skyrat/master_files/code/modules/clothing/towels.dm:44]: Just to make 10000% sure, this doesn't try to do anything fucky when it's in head shape?

**Comment by ghost** [comment]
Icons check out 👍 
I will say, next time for testing, uh-
Maybe a character that's not the same color as the item it's wearing? :p

**Comment by tf-4** [comment]
@Ebin-Halcyon

**Review by Ebin-Halcyon — APPROVED** [review]

**Comment by GoldenAlpharex** [comment]
> Icons check out 👍 I will say, next time for testing, uh- Maybe a character that's not the same color as the item it's wearing? :p

I just rolled with the first thing it gave me lol

**Pull request merged**